firebase/functions/src/index.ts

```diff
@@ -230,6 +230,7 @@ export const createChain = functions
         categories,
         published: true,
         chainAdmin: uid,
+        open: true,
```

Comment:
Are you going to run a migration for existing chains? Also, you might want to consider a slightly more descriptive name, or to add a few comments explaining what the `open` field means. Maybe `openNewMembers`, or `membersCanJoin`?

Reply:
I wasn't really sure how easy this is to do with Firebase, so I instead I decided to add a default value of `open: true` in frontend/src/util/firebase/chain.ts when fetching chains for those that don't have the field, to keep current behaviour. Does that seem OK?

Making the name more descriptive sounds good; I will change that.